modules/virt-define-http-liveness-probe.adoc

```diff
@@ -18,18 +18,26 @@ Define an HTTP liveness probe by setting the `spec.livenessProbe.httpGet` field
 .Sample liveness probe with an HTTP GET test
 [source,yaml]
 ----
+apiVersion: kubevirt.io/v1
+kind: VirtualMachine
+metadata:
+  annotations:
+  name: fedora-vm
+  namespace: openshift-cnv
 # ...
 spec:
-  livenessProbe:
-    initialDelaySeconds: 120 <1>
-    periodSeconds: 20 <2>
-    httpGet: <3>
-      port: 1500 <4>
-      path: /healthz <5>
-      httpHeaders:
-      - name: Custom-Header
-        value: Awesome
-    timeoutSeconds: 10 <6>
+  template:
+    spec:
+      livenessProbe:
+        initialDelaySeconds: 120 <1>
+        periodSeconds: 20 <2>
+        httpGet: <3>
+          port: 1500 <4>
+          path: /healthz <5>
+          httpHeaders:
+          - name: Custom-Header
+            value: Awesome
+        timeoutSeconds: 10 <6>
```

Comment:
**QE REVIEW NEEDED** Please verify the spacing in the YAML example.

Reply:
it is correct